modules/040-node-manager/openapi/values.yaml

```diff
@@ -14,7 +16,7 @@ properties:
         description: |
           Array of API servers addresses.
         x-examples:
-          - ["10.0.0.1:6443", "10.0.0.2:6443", "10.0.0.3:6443"]
+          - [ "10.0.0.1:6443", "10.0.0.2:6443", "10.0.0.3:6443" ]
```

Comment:
Why spaces? We don't use them anywhere else.

Reply:
Unfortunately, Goland format YAMLs for me this way. And this formatting is already present in some files. We could have example formatting rules for YAML, of a formatter in makefile.

I'll try to reduce this effect.

Reply:
Fixed.